Ecosystem Partners/Marketplace Guides/getting-started-with-waratek-locker.md

```diff
@@ -0,0 +1,153 @@
+
+###Audience
+
+CenturyLink Cloud Users desiring a secure Java environment.
+
+###Impact
+
+After reading this article, the user should be able to install Locker on a CenturyLink Cloud server.
+
+###Prerequisites
+
+* Access to the CenturyLink Cloud platform as an authorized user.
+
+###Postrequisites
+
+* None
+
+###Deploying Locker on a New Server
+
+Waratek Locker is available as a Blueprint for deployment on a new server.
+
+**Steps to deploy to New Server Blueprint**
+
+1. Locate the "Install Waratek Locker" Blueprint
+
+2. Click on the blueprint. Select "Deploy Blueprint" on the resulting screen
+
+3. Configure the Blueprint. Complete the information below:
+
+    1. **Enter password.**
+
+    2. **Choose your group.**
+
+    3. **Choose your network.**
+
+    4. **Choose Primary and Secondary DNS.**
+
+    5. **Choose the Server Type.**
+
+    6. **Choose Service Level.**
+
+    7. **Enter Server Name.**
+
+4. Review and Confirm the Blueprint
+
+    8. **Click "next: step 2"**
+
+    9. **Verify your configuration details.**
+
+5. Deploy the Blueprint
+
+    1. Once verified, click on the ‘deploy blueprint’ button. You will see the deployment details along with an email stating the Blueprint is queued for execution.
+
+    2. This will kick off the blueprint deploy process and load a page to allow you to track the progress of the deployment.
+
+* Monitor the Deployment Queue to view the progress of the blueprint. You can access the queue at any time by clicking the Queue link under the Blueprints menu on the main navigation drop-down.
+
+* Once the blueprint completes successfully, you will receive an email stating that the blueprint build is complete. Please do not use the application until you have received this email notification.
+
+###Steps to deploy to an existing server
+
+1. Deploy or Identify an existing server
+
+2. Locate the "Install Waratek Locker" blueprint in the Blueprint Library
+
+3. Select "Deploy Blueprint"
+
+4. Select the server to be deployed
+
+5. Select "next: step 2"
+
+6. Select "Deploy Blueprint"
+
+7. Monitor the Deployment Queue to view the progress of the blueprint. You can access the queue at any time by clicking the Queue link under the Blueprints menu on the main navigation drop-down.
+
+8. Once the blueprint completes successfully, you will receive an email stating that the blueprint build is complete. Please do not use the application until you have received this email notification.
+
+###Frequently Asked Questions
+
+For more information about Locker, click on the following links: 
+
+Locker User Guide ([http://www.waratek.com/resources/guides/locker/](http://www.waratek.com/resources/guides/locker/)) 
+
+Locker FAQs ([http://www.waratek.com/resources/guides/locker/frequently-asked-questions/](http://www.waratek.com/resources/guides/locker/frequently-asked-questions/)) 
+
```

Comment:
get rid of parentheses around links, and just use the text as the link.
You probably don't need the 'For more information...' line.  But up to you.